README.md

```diff
@@ -228,24 +265,30 @@ An example in Beats config might look like this:
   fields:
     type: trace
 ```
-The Logstash configuration would then look like this to check the 
+
+The Logstash configuration would then look like this to check the
 given field:
-```
+
+```none
 if ([fields][type] == "openlog") {
    Do something for type openlog
 ```
+
 But, in order to test the behavior with LFV you have to give it like so:
-```
+
+```json
 {
   "fields": {
     "[fields][type]": "openlog"
   },
 ```
```

Comment:
There appears to be something about this line (or some other backtick line above) that causes GitHub's diff view to go totally bonkers. Have you looked into what's going on?

Reply:
No, I did not look into this. I assume, it is due to the example being not valid json. I decided to switch the formatting for this block from `json` to `none`.

Reply:
Oh, that's odd. Seems like a GitHub bug. Glad you found it!